v21.1/build-a-nodejs-app-with-cockroachdb-sequelize.md

```diff
@@ -55,16 +55,8 @@ Open `app.js`, and edit the connection configuration parameters:
 
 <section class="filter-content" markdown="1" data-scope="cockroachcloud">
 
-- At the top of the file, uncomment the `const fs = require('fs');` line.
-
-    This line imports the `fs` Node module, which enables you to read in the CA cert that you downloaded from the {{ site.data.products.db }} Console.
-- Replace the value for `username` with the user you created earlier.
+- Replace the values for `username`, `host`, `port`, and `database` with values from the "Connection parameters" tab of "Connection info."
```

Comment:
For consistency with the rest of our Cloud docs:

s/`"Connection parameters" tab of "Connection info."`/`**Connection parameters** tab of the **Connection info** dialog.`

Reply:
Thanks!